Makefile.js

```diff
@@ -882,6 +882,11 @@ target.checkGitCommit = function() {
 
     // Only check non-release messages
     if (!semver.valid(commitMsgs[0])) {
+        if (commitMsgs[0].slice(0, commitMsgs[0].indexOf("\n")).length > 72) {
```

Comment:
If I'm not mistaken git will automatically add line break at 72 characters. If I'm correct this will always pass.